### ISSUE TITLE
[Research] OUI Compliance audit of the `vis_type_vislib` plugin

### ISSUE THREAD

**Issue by BSFishy** [opensearch-project/OpenSearch-Dashboards#4104]
Tracking issue for OUI compliance audit of the `vis_type_vislib` plugin.

Follow the steps in #3945 to complete the audit, then open another issue with findings. Once the work of performing the audit is done, this issue can be closed.

<details>
  <summary>Related files</summary>

  size (bytes) | file | notes
---|---|---
5.9K | [src/plugins/vis_type_vislib/public/vislib/lib/layout/_layout.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/vislib/lib/layout/_layout.scss) | 
2.0K | [src/plugins/vis_type_vislib/public/vislib/components/legend/_legend.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/vislib/components/legend/_legend.scss) | 
1.1K | [src/plugins/vis_type_vislib/public/vislib/lib/_alerts.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/vislib/lib/_alerts.scss) | 
1.1K | [src/plugins/vis_type_vislib/public/vislib/components/tooltip/_tooltip.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/vislib/components/tooltip/_tooltip.scss) | 
483 | [src/plugins/vis_type_vislib/public/vislib/visualizations/point_series/_labels.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/vislib/visualizations/point_series/_labels.scss) | 
421 | [src/plugins/vis_type_vislib/public/vislib/_vislib_vis_type.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/vislib/_vislib_vis_type.scss) | 
329 | [src/plugins/vis_type_vislib/public/vislib/lib/_handler.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/vislib/lib/_handler.scss) | 
238 | [src/plugins/vis_type_vislib/public/vislib/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/vislib/_index.scss) | 
212 | [src/plugins/vis_type_vislib/public/vislib/_variables.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/vislib/_variables.scss) | 
67 | [src/plugins/vis_type_vislib/public/vislib/lib/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/vislib/lib/_index.scss) | 
57 | [src/plugins/vis_type_vislib/public/vislib/visualizations/gauges/_meter.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/vislib/visualizations/gauges/_meter.scss) | 
25 | [src/plugins/vis_type_vislib/public/index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/index.scss) | 
21 | [src/plugins/vis_type_vislib/public/vislib/components/tooltip/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/vislib/components/tooltip/_index.scss) | 
20 | [src/plugins/vis_type_vislib/public/vislib/visualizations/point_series/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/vislib/visualizations/point_series/_index.scss) | 
20 | [src/plugins/vis_type_vislib/public/vislib/lib/layout/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/vislib/lib/layout/_index.scss) | 
20 | [src/plugins/vis_type_vislib/public/vislib/components/legend/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/vislib/components/legend/_index.scss) | 
19 | [src/plugins/vis_type_vislib/public/vislib/visualizations/gauges/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/vis_type_vislib/public/vislib/visualizations/gauges/_index.scss) | 

</details>

If this issue isn't already assigned to someone and you'd like to take on this work, please ping @BSFishy in the comments and I can assign it to you.
